market/connector/currencyConverter/demo/currencyConverter/dataclasses/currencyConverter/CurrencyConverterData.ivyClass

```diff
@@ -0,0 +1,8 @@
+CurrencyConverterData #class
+currencyConverter #namespace
+FromAmount Number #field
+FromAmount PERSISTENT #fieldModifier
```

Comment:
I assume the 'persistent' flags are not required here and came through the default settings.